docs/root/configuration/http/http_filters/cors_filter.rst

```diff
@@ -9,7 +9,15 @@ For the meaning of the headers please refer to the pages below.
 * https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS
 * https://www.w3.org/TR/cors/
 * This filter should be configured with the type URL ``type.googleapis.com/envoy.extensions.filters.http.cors.v3.Cors``.
-* :ref:`v3 API reference <envoy_v3_api_msg_config.route.v3.CorsPolicy>`
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.cors.v3.CorsPolicy>`
+
+
+.. note::
+  This filter is only used to respond to preflight ``OPTIONS`` requests. Any non-``OPTIONS`` requests or illegal ``OPTIONS``
```

Comment:
Doesn't it also add headers on the response pass for regular methods like GET?

Reply:
Yeah, a GET request with legal `origin` header will be inserted some headers on the response phase.

Seems a bug? Is there any reason to do the inserts for the GET requests?

Reply:
I re-checked the code and related rfc doc and updated the note. Thanks for calling this out👍 . I almost made a big mistake.